docs/source/extension/extension_points.rst

```diff
@@ -419,6 +419,31 @@ declaring default keyboard shortcuts for a command:
 
 Shortcuts added to the settings system will be editable by users.
 
+From Jupyterlab version 3.1 onwards, it is possible to execute multiple commands with a single shortcut. 
+This requires you to define a keyboard shortcut for apputils:run-all-enabled command:
```

Comment:
```suggestion
This requires you to define a keyboard shortcut for ``apputils:run-all-enabled`` command:
```

Reply:
Nice catch! The change is applied. Thank you.